src/plugins/intel_gpu/src/kernel_selector/kernels/reduce/reduce_kernel_b_fs_yx_fsv16.cpp

```diff
@@ -216,14 +216,30 @@ JitConstants ReduceKernel_b_fs_yx_fsv16::GetJitConstants(const reduce_params& pa
         }
     }
 
+    // MIN/MAX mode should handle feature remainder in case reduce axes includes feature
+    if (params.reduceMode == ReduceMode::MIN || params.reduceMode == ReduceMode::MAX) {
+        if (count(params.reduceAxes.begin(), params.reduceAxes.end(), 1) > 0) {
+            if (params.inputs[0].Feature().v % 16 != 0) {
+                jit.AddConstant(MakeJitConstant("HANDLE_FEATURE_REMAINDER", 1));
+            }
+        }
+    }
+
     return jit;
 }
 
+bool ReduceKernel_b_fs_yx_fsv16::Validate(const Params& p, const optional_params& options) const {
+    if (!ReduceKernelBase::Validate(p, options))
+        return false;
+
+    return true;
+}
```

Comment:
Why this was needed though there is no additional check here?

Reply:
deleted